README.markdown

```diff
@@ -436,16 +436,27 @@ scripts.
 
 __4.12. Q. How can I check scripts written for different versions of Ruby?__
 
-A. Install a Ruby version manager such as [rvm][rvm] or [rbenv][rbenv],
-activate the environment for the relevant version of Ruby, and install in it
-the checkers you want to use.  Set `g:syntastic_ruby_checkers` accordingly in
-your `vimrc`, and run [Vim][vim] from the virtual environment.
-
-If you're starting Vim from a desktop manager rather than from a terminal you
-might need to write wrapper scripts around your checkers, to activate the
-virtual environment before running the actual checks.  Then you'll need to
-point the relevant `g:syntastic_ruby_<checker>_exec` variables to the wrapper
-scripts.
+A. Install a Ruby version manager such as [rvm][rvm],  [rbenv][rbenv],
+[chruby][chruby] or [ry][ry], activate the relevant version of Ruby, and
```

Comment:
> Does using `exe` or `exec` actively mangling exported `PATH`, `GEM_*` or `RUBY_*` vars?

No, but several internals rely on the result of `executable()`.  This is not important in simple cases, but it can get really messy in more complicated setups (think `node`).  There has been an example of this causing problems a few weeks ago.

> Does it change the behaviour of system symlinks?

No, but symlinks take effect globally, not per project.

> Otherwise env vars just flow through vim as is to ruby and the checkers, and symlinks are just symlinks.

Nope.  Vim may mimic the behaviour of a shell in many respects, but it isn't a shell itself.  Perhaps surprisingly, `system()` in Vim is a really complicated mess.  This is well-known for Python's virtualenv, where various Vim plugins try to mess with the environment and run into a wall. _shrug_